### DISCUSSION TITLE
Matrix and XMPP: Thoughts on Improving Messaging Protocols – Part 1

### DISCUSSION THREAD

**Discussion by mremond** [processone/ejabberd#4289] [Announcements]
I have shared some thoughts about XMPP, Matrix and I discussed what the protocols can learn from each others.

The blog post is available here: https://www.process-one.net/blog/matrix-and-xmpp-thoughts-on-improving-messaging-protocols-part-1/

**Comment by licaon-kter** [comment]
_I see Matrix, I downvote_

Jokes aside, _there's a XEP for that_, right? https://xmpp.org/extensions/xep-0289.html

**Reply by mremond** [reply]
Yes, right. This is a bit different of what I had in mind (More of a kind of automatic cache), but yes, this is interesting.
